sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -1066,6 +1066,26 @@ case class CreateFunction(
     copy(child = newChild)
 }
 
+/**
+ * The logical plan of the CREATE FUNCTION command for SQL Functions.
+ */
+case class CreateUserDefinedFunction(
```

Comment:
shall we follow `UnsetNamespacePropertiesCommand`? i.e. extends `UnaryRunnableCommand` and implement `def run`, so that we only need one logical plan: the command itself.

Reply:
This is a good suggestion. I've created a follow up ticket here: [SPARK-50698](https://issues.apache.org/jira/browse/SPARK-50698)